src/helpers/accessibility.ts

```diff
@@ -0,0 +1,43 @@
+import { ReactTestInstance } from 'react-test-renderer';
```

Comment:
In TL/Jest-Native repo there is a related [PR for `toBeVisible` Jest matcher](https://github.com/testing-library/jest-native/pull/73/files). It seems that they are related and check the same concept of "being visible to the user". From the other PR I've noticed that they also check for `opacity: 0` style

Reply:
oh great catch! I'll add it to the list then